test/actions/ScrummyAPI.js

```diff
@@ -5,6 +5,10 @@ import sinon from 'sinon';
 import thunkMiddleware from 'redux-thunk';
 const mockStore = configureStore([]);
 
+test.beforeEach(() => {
+  global.ga = () => {};
```

Comment:
Let's move this to https://github.com/fourkitchens/scrummy-react-dom/blob/master/test/helpers/dom.js#L13. That way you don't have to do it twice. You'll be able to delete the `beforeEach()`